### ISSUE TITLE
bad fountain shape rendering

### ISSUE THREAD

**Issue by hehehe92** [osmandapp/OsmAnd#19127]
### Description

![Screenshot_20240216-210100_Firefox](https://github.com/osmandapp/OsmAnd/assets/158172547/d58d6a48-8a49-4136-872c-941372b06097)

### Steps to reproduce

open:
Водограй
Bulevardul Unirii, București
Розташування: geo:44.42681,26.10175?z=21
https://osmand.net/map?pin=44.42681,26.10175#21/44.42681/26.10175

### Actual result

above

### Expected result

like below

### Your Environment (required)

```
OsmAnd Version:
Android version: 10
Device model: Galaxy S9
```

**Comment by mikehgentry** [comment]
Presumably #9821 or one several similar bugs: as I understand it the precision breaks down at a certain point, and it really isn't fixable at present.

**Comment by vshcherb** [comment]
Yes it's not fixable cause precision is part of the maps and it will break backward compatibility with many old devices.
